contents/blog/brand.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2023-06-25
+title: How to brand your startup
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: CEO diaries
+---
+
+The world would be a lot more fun if most startups didn't seem to have experienced a personality vasectomy.
+
+I believe humans have evolved to conform - when we all lived in the wilderness, staying alive required you to fit in with a tribe, after all. Fast forward that background to today and, amongst other things (!), the majority of the world's software companies feel the same as each other.
+
+## There are good reasons for this, you idiot
+
+I bet there are. You want to get the Minimum Viable Product built and validated.
+
+You need to make money, you're going to fail without it.
+
+Creating a pretty website won't help with that.
+
+## But... will it?!
+
+Maybe!
+
+When we knew that PostHog was something we wanted to launch, we questioned how we'd get the most users signed up.
+
+The product on launch day was going to be free, but we still asked ourselves: "what would stop me from using this thing?"
+
+Despite being free, users would be wary of wasting their time trying to deploy a badly supported, broken piece of open source software, or worse - it not handling their data securely. These are both things you'd associate with the software produced by a 2 person startup.
+
+Before a user even gets into the software or glances at the code first, the majority will turn away _at_ the website, the docs and the readme.
+
+As a counter example, an obviously templated or one-pager style website:
+
+* signals the product may not be secure, a lack of demonstrated competence in the website shows a lack of attention to detail - if a startup doesn't care about having their own design, will they care about security?
+* signals the team behind the product isn't very strong, and won't improve quickly over time, or become a "must use" tool - those startup launches that everyone hears about? They've put energy into their appearance.
+* signals the product may not exist in a few more weeks, which would be a waste of time - when it's obvious the website was thrown together in a day - if the team don't look invested in their company, why would you put your own time in?
+
+I admit this is frustrating - it is extra time and effort into something "superficial" - just remember, you are providing your product to human beings, take advantage of their pattern recognition (it also helped them survive in the wild).
+
+## Your company needs to stand for something
+
+Take two examples:
+
+* Open source product analytics
+* We equip every developer to build successful products
+
+There is a tradeoff between the two.
+
+The first is very clear. We started with it when this is what we had.
```

Comment:
'We started with this when this is what we had' confused me. you mean, like...

This was what our company stood for when TKTKTK.

something like that would be clearer